grpc-proxy/proxy-server/main.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
Discussed on Slack and agreed that a meaningful test would probably require a Temporal instance unlike the other samples in this repo. Given that, an integration test should be added to sdk-go rather than here.